### PR TITLE
test:add test cases for func in opt/config/ulimit

### PR DESCRIPTION
**Pull request by WillSmisi**
Signed-off-by: WillSmisi <764947976@11.com>

### Ⅰ. Describe what this PR did
test:add test cases for func in opt/config/ulimit

### CODE REVIEW

File: apis/opts/config/ulimit_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+)
+
+func TestUlimit_Set(t *testing.T) {
+	type fields struct {
+		values map[string]*units.Ulimit
+	}
+	type args struct {
+		val string
+	}
+	tests := []struct {
```
Comment (Ace-Tang) [inline apis/opts/config/ulimit_test.go]: I was curious why not use
```
struct{
                name    string
		fields     values map[string]*units.Ulimit
		args     string
		wantErr bool
	}
```

**Comment by chuanchang** [comment]
@WillSmisi Thank you for your contribution, in fact, it exists a PR #2328 to add test cases and go through uncovered code path, please make sure it's not duplicate testing firstly.

File: apis/opts/config/ulimit_test.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/docker/go-units"
+)
+
+func TestUlimit_Set(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (WillSmisi) [reply apis/opts/config/ulimit_test.go]: Thank you for the suggestion. I just found the same style of naming the functions in other files. I will take the suggestion into my heart.

File: apis/opts/config/ulimit_test.go (hunk 1 above)
Comment (WillSmisi) [reply apis/opts/config/ulimit_test.go]: Ok, thank you for the suggestion. Maybe it's clearer by using the way that you suggest.

**Comment by WillSmisi** [comment]
> @WillSmisi Thank you for your contribution, in fact, it exists a PR #2328 to add test cases and go through uncovered code path, please make sure it's not duplicate testing firstly.

@chuanchang Thank you for telling me the message. I'm so sorry about that. I didn't discover it when I submit the PR. I'll pay attention to it in due course. Now i close the PR. Thank you.
